CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 - Refactor the models file generation to use the artifacts from the schemas.
 - Add `build_json` and `build_yaml` interfaces which can be used to produce a package with the models.
+- Ensure a package built with `build_json` or `build_yaml` is contained within a project directory.
```

Comment:
My preference would be to state what the change is, e.g. `Changed output of `build_json` and `build_yaml` to be contained within a project directory.` As a reader I wouldn't be sure what ensured meant, is it a check that is run?

Reply:
Makes sense. Addressed ✅